packages/quickwit/v0.8.1+4/package.yaml

```diff
@@ -0,0 +1,158 @@
+# yaml-language-server: $schema=https://glasskube.dev/schemas/v1/package-manifest.json
+name: quickwit
+shortDescription: Cloud-native search engine for observability
+longDescription: |
+  > _**PLEASE READ:** If you previously installed this package as a `ClusterPackage`, please remove and install it as a
+  > namespace-scoped `Package` instead._
+
+  Quickwit is the first engine to execute complex search and analytics queries directly on cloud storage with
+  sub-second latency. Powered by Rust and its decoupled compute and storage architecture, it is designed to be
+  resource-efficient, easy to operate, and scale to petabytes of data.
+
+  Quickwit is a great fit for log management, distributed tracing, and generally immutable data such as conversational
+  data (emails, texts, messaging platforms) and event-based analytics.
+
+  Make sure that the object storage user has all permissions (e.g. `"s3:*"`) for the configured bucket.
+  For more information make sure to check out the Quickwit documentation.
+scope: Namespaced
+defaultNamespace: quickwit
+iconUrl: https://avatars.githubusercontent.com/u/98504233
+helm:
+  repositoryUrl: "https://helm.quickwit.io"
+  chartName: quickwit
+  chartVersion: "0.5.16"
+  values:
+    environment: {}
+    config:
+      storage:
+        s3: {}
+    ingress:
+      enabled: false
+      annotations:
+        cert-manager.io/cluster-issuer: letsencrypt
+      hosts:
+        - host: ""
+          paths:
+            - path: /
+              pathType: ImplementationSpecific
+      ingressClassName: nginx
+      tls:
+        - hosts: [ ]
+valueDefinitions:
+  defaultIndexRootUri:
+    type: text
+    metadata:
+      description: >
+        The default index URI is a S3 bucket which usually looks like this: "s3://<bucket-name>/<optional-base-path>"
```

Comment:
not related to this PR, but I noticed that this no longer works correctly because of https://github.com/glasskube/glasskube/pull/974

Reply:
Good point, I tried wrapping it in `...` which would work for the UI, but not on the CLI because it's not resolved yet:

```
go run cmd/glasskube/main.go configure test --kind package --namespace ns-1

quickwit has 8 values for configuration.

defaultIndexRootUri
The default index URI is a S3 bucket which usually looks like this: `s3://<bucket-name>/<optional-base-path>`

Old value: s3://local-christoph-my-org-2-quickwit4
Keep? (Y/n) ^Cexit status 1
```

Reply:
I think that's fine, we can fix the CLI later.